api_tests/users/views/test_user_draft_registration_list.py

```diff
@@ -107,22 +116,19 @@ def test_draft_with_registered_node_does_not_show_up_in_draft_list(
         data = res.json['data']
         assert len(data) == 0
 
-    def test_draft_with_deleted_registered_node_shows_up_in_draft_list(
+    def test_draft_with_deleted_registered_node_does_not_show_up_in_draft_list(
```

Comment:
This shouldn't have changed. Deleting the registered node when it's rejected by admins is part of what makes the draft editable.